# ProteomicsSIP_analysis

## data and code for analysis of Sipros based Proteomic SIP experiments

### DataAnalysis.ipynb
- This notebook presents some code used to analyze data generated from a Proteomic SIP experiment that tracked incorporation of 13C-labeled amino acids into a sampled bacterioplankton consortia. Results were reported in:

    - Bryson, Samuel, Zhou Li, Jennifer Pett-Ridge, Robert L. Hettich, Xavier Mayali, Chongle Pan, and Ryan S. Mueller. 2016. “Proteomic Stable Isotope Probing Reveals Taxonomically Distinct Patterns in Amino Acid Assimilation by Coastal Marine Bacterioplankton.” mSystems 1 (2).* 


- Data analysis needed to allow exploration of community structure at different taxonomic ranks, protein abundance (also at different taxonomic ranks), and protein labeling according to two metrics developed to characterize substrate incorporation.

    1- Average Enrichment - The Average 13C% of labeled proteins/peptides; this metric conveys information about the specialization of a particular taxa or population for a specific substrate.
    
    2- Label Frequency - The proportion of an identified proteome that is labeled (i.e. 13C% above background); this metric conveys information about a particular taxa or population's protein synthesis activity.



- Notebooks for later work that built off this analytical approach may be uploaded later


- Additional papers that utilized the analytical approach here (using the two metrics- Average Enrichment and Label Frequency) are listed below. 
    - Bryson, Samuel, Zhou Li, Francisco Chavez, Peter K. Weber, Jennifer Pett-Ridge, Robert L. Hettich, Chongle Pan, Xavier Mayali, and Ryan S. Mueller. 2017. “Phylogenetically Conserved Resource Partitioning in the Coastal Microbial Loop.” The ISME Journal 11 (12): 2781–92.

    - Kieft, Brandon, Zhou Li, Samuel Bryson, Robert L. Hettich, Chongle Pan, Xavier Mayali, and Ryan S. Mueller. 2021. “Phytoplankton Exudates and Lysates Support Distinct Microbial Consortia with Specialized Metabolic and Ecophysiological Traits.” Proceedings of the National Academy of Sciences of the United States of America 118 (41). 

